namespace.go

```diff
@@ -179,3 +179,116 @@ func makeName(name string, unit Unit) string {
 
 	return fmt.Sprintf("%s_%s", name, unit)
 }
+
+func (n *Namespace) NewDefaultHttpMetrics() []*HttpMetric {
+	return n.NewHttpMetricsWithOpts(HttpHandlerOpts{
+		DurationBuckets:     defaultDurationBuckets,
+		RequestSizeBuckets:  defaultResponseSizeBuckets,
+		ResponseSizeBuckets: defaultResponseSizeBuckets,
+	})
+}
+
+func (n *Namespace) NewHttpMetrics(durationBuckets, requestSizeBuckets, responseSizeBuckets []float64) []*HttpMetric {
+	return n.NewHttpMetricsWithOpts(HttpHandlerOpts{
+		DurationBuckets:     durationBuckets,
+		RequestSizeBuckets:  requestSizeBuckets,
+		ResponseSizeBuckets: responseSizeBuckets,
+	})
+}
+
+func (n *Namespace) NewHttpMetricsWithOpts(opts HttpHandlerOpts) []*HttpMetric {
+	if _, ok := n.labels["handler"]; !ok {
+		panic("label \"handler\" must be provided in the namespace")
```

Comment:
Was this part of the original interface?

Reply:
Ok, so it looks like we need a specialized constructor that creates this for the particular handler. Looking at the example, it should not be required to construct the namespace with a specific label. That should be part of creating the handler metric.